Formula/buildpulse-test-reporter.rb

```diff
@@ -0,0 +1,27 @@
+class BuildpulseTestReporter < Formula
+  desc "Connect your CI to BuildPulse to detect, track, and rank flaky tests"
+  homepage "https://buildpulse.io"
+  url "https://github.com/buildpulse/test-reporter/archive/refs/tags/v0.15.0.tar.gz"
+  sha256 "5119c189341bec12531fa85ac504a3c88b5ca08f090f2ee92d6cef667d66684d"
+  license "MIT"
+  head "https://github.com/buildpulse/test-reporter.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    goldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+      -X main.Commit=#{tap.user}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: goldflags), "./cmd/test-reporter"
```

Comment:
I'm new to defining Homebrew formulas, so this might be a silly question: Can the formula rely on the [released binary](https://github.com/buildpulse/test-reporter/releases/download/v0.15.0/test-reporter-darwin-amd64) instead of building it from scratch? 

I ask because the binary's build configuration may change over time, and it would be nice to only have to [maintain that configuration in the source repo](https://github.com/buildpulse/test-reporter/blob/v0.15.0/.goreleaser.yml) instead of also maintaining it in Homebrew/homebrew-core. Does that make sense?

Reply:
Formulae in this repository must always be built from source. If the build process changes in the future, the formula will be updated to reflect that – just like all the other formulae here.

Reply:
If it helps, you don't have any responsibility for this formula -- we'll take care of keeping it up to date. Of course, we appreciate any help you're willing to provide on this 🙂

Reply:
> Formulae in this repository must always be built from source. If the build process changes in the future, the formula will be updated to reflect that – just like all the other formulae here.

Ah, thanks for that insight. 👍 :bow: